README.rst

```diff
@@ -0,0 +1,80 @@
+.. image:: logo.png
+
+Rohmu is a Python library for building backup tools for databases
+providing functionality for compression, encryption and transferring
+data between the database and an object storage. Rohmu supports main
+public clouds such as GCP, AWS and Azure for backup storage. Rohmu is
+used in various backup tools such as
+`PGHhoard <https://github.com/aiven/pghoard>`__ for PostgreSQL,
+`MyHoard <https://github.com/aiven/myhoard>`__ for MySQL and
+`Astacus <https://github.com/aiven/astacus>`__ for M3 and ClickHouse and
+other databases.
+
+Features
+========
+
+-  Supported object storages: Azure, GCP, S3, Swift (OpenStack), local
+   file storage and SFTP.
+-  Supported compression algorithms: Snappy,
+   `zstd <https://github.com/facebook/zstd>`__ and
+   `lzma <https://docs.python.org/3/library/lzma.html>`__.
+
+Requirements
+============
+
+Rohmu requires Python >= 3.6+. For Python libary dependencies, have a
+look at
+`requirements.txt <https://github.com/aiven/rohmu/blob/main/requirements.txt>`__.
+
+Usage example
+=============
+
+**Add usage example here**
+
+For real-world usage you can have a look at `how Rohmu is used in
+PGHoard <https://github.com/aiven/pghoard/blob/main/pghoard/basebackup.py>`__.
+
+Development
+===========
+
+**TODO**
+
+License
+=======
+
+Rohmu is licensed under the Apache license, version 2.0. Full license
+text is available in the `LICENSE <LICENSE>`__ file.
+
+Please note that the project explicitly does not require a CLA
+(Contributor License Agreement) from its contributors.
+
+Trademarks
+==========
+
+PostgreSQL, MySQL, M3 and ClickHouse are trademarks and property of
+their respective owners. All product and service names used in this
+website are for identification purposes only and do not imply
+endorsement.
+
+Credits
+=======
+
+Rohmu was created by, and is maintained by,
```

Comment:
Could you please remove redundant commas here?

Reply:
I did not check line by line, but those commas where in the previous version of the test I guess. Hopefully nothing changes expect for this?

Reply:
Oh no, this is done by a crappy converter. Going to check for more of those.

Reply:
Now checked line by line. Can't see any differences between the old and the new one.